plugins/inputs/mysql/mysql.go

```diff
@@ -869,7 +869,23 @@ func (m *Mysql) gatherGlobalStatuses(db *sql.DB, servtag string, acc telegraf.Ac
 			if err != nil {
 				acc.AddError(fmt.Errorf("error parsing mysql global status %q=%q: %w", key, string(val), err))
 			} else {
-				fields[key] = value
+				// v2.ConvertGlobalStatus can return multiple values in the form of a map, when needed
+				// 	e.g. wsrep_evs_repl_latency
+				//  mysql> show global status like 'wsrep_evs_repl_latency';
+				//  +------------------------+---------------------------------------------------+
+				//  | Variable_name          | Value                                             |
+				//  +------------------------+---------------------------------------------------+
+				//  | wsrep_evs_repl_latency | 0.000336823/0.000518414/0.000700005/0.000181591/2 |
+				//  +------------------------+---------------------------------------------------+
+				//  1 row in set (0.00 sec)
```

Comment:
Please remove this extensive documentation as we might want to add more "complex" fields in the future...

```suggestion
```

Reply:
I was in doubt myself if this would be a good place for that type of comment. Removed.